src/views/fragments/IntroductionContent/index.jsx

```diff
@@ -0,0 +1,61 @@
+import { useNavigate,useLocation } from "react-router-dom";
+import ROUTE_NAMES from "../../../constants/routeNames";
+import AVATARS from "../../../constants/avatars";
+import React from "react";
+
+import {
+    StyledBackButton,
+    IntroductionContentWrapper,
+    Paragraph,
+    ParagraphContainer,
+    Intro,
+    AvatarImage,
+    ParagraphHeader,
+  } from "./styled.components";
+
+
+  const { QUESTIONS } = ROUTE_NAMES
+  
+  const IntroductionContent =() =>{
+    const { state } = useLocation();
+    const navigate = useNavigate();
+    const usernames = state.usernames;
+    const selections = state.selections;
+
+    const handleNext = () => {
+      navigate(QUESTIONS, {state:{usernames,selections}});
+    };
+
+    
+
+    return(
+        <>
+        <IntroductionContentWrapper>
+        <Intro>
```

Comment:
wrap the images in an appropriate wrapper, the `<Intro />` component is used for displaying the header and not to position the avatars